app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,7 @@
             android:name="org.apache.http.legacy"
             android:required="false" />
 
-        <activity android:name=".activities.ReportMapActivity">
+        <activity android:name="app.vdh.org.vdhapp.feature.report.presentation.map.activity.ReportMapActivity">
```

Comment:
not sure this fully qualified name is needed, you could just use `.feature.report.presentation.map.activity.ReportMapActivity`